lib/riemann/tools.rb

```diff
@@ -65,6 +65,7 @@ def report(event)
       end
       
       event = event.merge(attributes)
+      event[:time] = Integer(event[:time]) if event.has_key? :time
```

Comment:
I'd like to avoid this kind of automatic coercion--don't want folks accidentally parsing `nil` or `""` as 0, etc. Can you move it into your log parser?